packages/components/src/components/app-card/hooks/use-hover.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export const useHover = refSetter => {
```

Comment:
We already have this hook in `packages/components/src/hooks/use-hover.js`

Reply:
Ah, perhaps we could pass an extra param to the existing hook which will set to which events it should listen.

Reply:
I've added an extra param `should_prevent_bubbling` which will set the events accordingly.